tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -147,6 +149,35 @@ func createTempFile() string {
 	return fileName
 }
 
+// executing test
+func executeTest(flags []string, m *testing.M) (successCode int) {
+	for i := 0; i < len(flags); i++ {
+		if err := mountGcsfuse(flags[i]); err != nil {
+			log.Printf("mountGcsfuse: %v\n", err)
+			os.Exit(1)
+		}
+
+		log.Printf("Test log: %s\n", logFile)
```

Comment:
why are you printing this?

Reply:
For getting logFile path

Reply:
why do you need to log the log file path. Isn't entire mounting command logged by default?